frontend/tailwind.config.js

```diff
@@ -1,8 +1,76 @@
 /** @type {import('tailwindcss').Config} */
-export default {
-  content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
+module.exports = {
+  darkMode: ['class'],
+  content: [
+    './pages/**/*.{ts,tsx}',
+    './components/**/*.{ts,tsx}',
+    './app/**/*.{ts,tsx}',
+    './src/**/*.{ts,tsx}',
```

Comment:
Why are we explicitly adding these  ? `./src/**/*.{js,ts,jsx,tsx}` this regex should ideally cover all the above newly added contents

Reply:
Same here file is changed after installation.

Reply:
Please revert :)

Reply:
Sure, I will revert this file and index.css file because we can add variables when we needed.